website/content/docs/k8s/installation/vault/connect-ca.mdx

```diff
@@ -11,6 +11,8 @@ description: >-
 Consul allows using Kubernetes auth methods to configure Connect CA.
 This allows for automatic token rotation once the renewal is no longer possible.
 
+In order to create Vault auth roles for the Consul servers for this feature, ensure that the Vault Kubernetes auth method is enabled as described in [Vault Kubernetes Auth Method](/docs/k8s/installation/vault#vault-kubernetes-auth-method).
+
```

Comment:
We don't automate the creation of the Vault Auth roles unfortunately since that would require escalated privileges. This sentence ensures that the Vault K8s auth method is enabled to follow the subsequent steps in the doc.

Reply:
Gotcha, so maybe something like this ???

```suggestion
The Vault as the Service Mesh Certificate Provider on Kubernetes feature enables integration with Vault. To leverage Vault for  management of Vault auth roles for the Consul servers, ensure that the Vault Kubernetes auth method is enabled as described in [Vault Kubernetes Auth Method](/docs/k8s/installation/vault#vault-kubernetes-auth-method)
```